specs/interop/predeploys.md

```diff
@@ -417,6 +421,46 @@ dependency set called `dependencySet()`. This function MUST return the array of
 `L1Block` MUST also provide a public getter to get the dependency set size called `dependencySetSize()`. This function
 MUST return the length of the dependency set array.
 
+## OptimismMintableERC20Factory
+
+| Constant | Value                                        |
+| -------- | -------------------------------------------- |
+| Address  | `0x4200000000000000000000000000000000000012` |
+
+### OptimismMintableERC20
+
+The `OptimismMintableERC20Factory` creates ERC20 contracts on L2 that can be used to deposit
+native L1 tokens into (`OptimismMintableERC20`). Anyone can deploy `OptimismMintableERC20`s contracts.
+
+Each `OptimismMintableERC20` contract created by the `OptimismMintableERC20Factory`
+allows for the `L2StandardBridge` to mint
+and burn tokens, depending on whether the user is
+depositing from L1 to L2 or withdrawing from L2 to L1.
+
+### Creation method
+
+The `OptimismMintableERC20Factory` is updated to use `CREATE3` instead of `CREATE2`.
+This will remove the compiler's dependency on address computation.
+
+The salt SHOULD depend on the inputs:
```

Comment:
A quick sentence for rationale here would be good, like these are used to ensure that there is a single unique token address per set of ERC20 metadata